source/extensions/filters/http/jwt_authn/jwks_async_fetcher.cc

```diff
@@ -11,8 +12,22 @@ namespace HttpFilters {
 namespace JwtAuthn {
 namespace {
 
-// Default cache expiration time in 5 minutes.
-constexpr int PubkeyCacheExpirationSec = 600;
+// Default cache expiration time in 10 minutes.
+constexpr std::chrono::seconds DefaultCacheExpirationSec{600};
+
+// Number of seconds to refetch before a cached jwks is expired.
+constexpr std::chrono::seconds RefetchBeforeExpiredSec{5};
+
+// Number of seconds to refetch after a failed fetch.
+constexpr std::chrono::seconds RefetchAfterFailedSec{1};
```

Comment:
this gets renamed to `DefaultRefetchAfterFailedSec` ?

And 

`// Default number of seconds before we refetch after a failed fetch` something on these lines?

Reply:
Done